### DISCUSSION TITLE
Custom Jump word keymap

### DISCUSSION THREAD

**Discussion by radvil** [LazyVim/LazyVim#3220] [Q&A] [answered]
Hi guys, after the latest update that removed vim-iluminate for the core spec, is there is any workaround to customize the jump keymap?. I am not using `]]` and `[[` as my jump keymap.

**Comment by folke** [comment, marked as answer]
I just moved the keymaps to lsp keymaps. See the doc below on how you can customize it:

https://www.lazyvim.org/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps

These are the relevant ones:

```lua
      { "]]", function() LazyVim.lsp.words.jump(vim.v.count1) end, has = "documentHighlight", desc = "Next Reference" },
      { "[[", function() LazyVim.lsp.words.jump(-vim.v.count1) end, has = "documentHighlight", desc = "Next Reference" },
```

**Reply by radvil** [reply]
Thanks..
